.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -0,0 +1,38 @@
+name: E2E tests ondemand
+
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        description: 'Target branch to run E2E tests over'
+        required: true
+        # TODO: @andriisoldatenko revisit it
+        default: 'release-1.1'
+
+permissions:
+  checks: write
+
+jobs:
+  run-in-k8s-1-21:
+    name: Run in Kubernetes 1.21
```

Comment:
why 1.21, according to our support chart https://docs.dynatrace.com/docs/setup-and-configuration/technology-support/support-model-for-kubernetes#:~:text=May%201%2C%202024-,1.21,-4.8 
1.21 is out of support since November

Reply:
@luhi-DT fixed.